.pre-commit-config.yaml

```diff
@@ -3,3 +3,48 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
+        exclude: |
+            (?x)^(
+                src/home_robot/home_robot/agent/perception/detection/lseg/|
+                src/home_robot/home_robot/perception/detection/detic/
+            )
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.11.5
+    hooks:
+      - id: isort
+        args: ["--profile", "black", "--filter-files"]
+        exclude: |
+            (?x)^(
+                src/home_robot/home_robot/agent/perception/detection/lseg/|
+                src/home_robot/home_robot/perception/detection/detic/|
+                src/home_robot_hw/
```

Comment:
Why are we excluding all of home_robot_hw from isort, mypy, flake8? I understand why we did this for the rest.

Reply:
Because of #36, we don't want to cause large merge conflicts.
Now #36 is merged, I will rebase and remove the exclusion.